recipes/ctags/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "ctags" %}
+{% set version = "5.8" %}
+{% set sha256 = "0e44b45dcabe969e0bbbb11e30c246f81abe5d32012db37395eb57d66e9e99c7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://prdownloads.sourceforge.net/ctags/ctags-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
+
+requirements:
+  build:
+    - toolchain
```

Comment:
That's kind of gross, but okay, done (https://github.com/conda-forge/staged-recipes/pull/3207/commits/fafee491ab2cd983e42b35328555b67504fe61df).